neuraxle/base.py

```diff
@@ -115,6 +130,11 @@ def append(self, current_id, data_input, expected_output):
         self.data_inputs.append(data_input)
         self.expected_outputs.append(expected_output)
 
+    def concat(self, data_container: DataContainer):
```

Comment:
is it possible to have an IterableDataContainer ? Looking for something like this right now : `iterable_utils.concat_iterables(iterable_data_container1, iterable_data_container2)`

Reply:
The only problem with that is that if a step needs a numpy array as an input this will make things weird and it won't be numpy that we send to the step.

Reply:
I think I have a solution !!

Reply:
https://docs.python.org/3/library/itertools.html#itertools.chain that's it !!!

Reply:
to be done in another pr @guillaume-chevalier resolving

Reply:
I like your idea, but it doesn't resolve the problem of using numpy arrays. Perhaps this new type of data container could be used only in joiners of minibatch.

Reply:
yes, add issue plz

Reply:
bro numpy is already iterable the itertools work